src/pages/Ratingresult.js

```diff
@@ -38,14 +38,15 @@ const CircleStyled = styled.div`
   align-items: center;
   justify-content: center;
   font-size: 32pt;
-  background-color: var(--secondary-pink);
+  background-color: var(--primary);
+  box-shadow: 0 4px 6px rgba(0, 0, 0, 0.4);
```

Comment:
Ich glaube, es macht Sinn, in einem Kosmos zu bleiben, sonst hast du immer hex. Ich habe es aber genau so gemacht, weil ich nicht weiß, wie es besser geht ;)  Wo ich drüber nachdenke: Man kann es bestimmt einfach umrechnen lassen #google #tool